pom.xml

```diff
@@ -834,5 +840,30 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
```

Comment:
It might be best if the e2e-test profile told Surefire not to run:

```xml
    <plugins>
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>]
        <configuration>
          <skipTests>true</skipTests>
        </configuration>
      </plugin>
    </plugins>
```